cobalt/h5vcc/h5vcc_crash_log.idl

```diff
@@ -108,4 +108,19 @@ interface H5vccCrashLog {
   // Sets a persistent Watchdog setting that determines whether or not a
   // Watchdog violation will trigger a crash.
   void setPersistentSettingWatchdogCrash(boolean can_trigger_crash);
+
+  // Appends a string event to a ring buffer. These log events can be appended
+  // from JS code. When watchdog violation is created, a snapshot of
+  // that buffer is attached to a violation. Identical sequential events
+  // are de-duplicated.
+  // Max event length is 256. Ring buffer size is 128.
```

Comment:
@briantting this equates to an ~32K max ring buffer size. WDYT? Is that too much?

Reply:
I spoke with Kaido about this. He has no concerns about 32K.

@alunev will this 32K be RAM and/or Disk? When we write violations to disk, will we also clear RAM?

Reply:
this is RAM and disk. When we write violations to disk we don't clear RAM.